tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "target": "es2015",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "lib": [
+            "es2017",
```

Comment:
nitpick 2: puppeteer supports evaluating code with `dom.iterable` and `es2018.asynciterable`

Reply:
I think the type libs should be put into Actor `tsconfig.json`, unless or until we start writing Typescript tests and examples in the SDK. Good point though, we should mention this in the Typescript README for now, so it will not be forgotten.

Reply:
Done. The doc file is called `docs/guides/typescript_actor.md`. We'll need to add an example of a basic TS Actor soon.